docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityService.php

```diff
@@ -105,23 +113,24 @@ class PostFacilityService extends PostFacilityBase {
    *   The PostAPI service.
    * @param \Drupal\Core\Render\Renderer $renderer
    *   Core renderer.
-   * @param Drupal\Core\File\FileUrlGeneratorInterface $file_url_generator
-   *   File URL generator.
+   * @param \Drupal\Core\File\FileSystemInterface $file_system
+   *   File system service.
+   * @param \Drupal\file\FileRepositoryInterface $file_repository
+   *   File repository service.
    */
-  public function __construct(ConfigFactoryInterface $config_factory, EntityTypeManagerInterface $entity_type_manager, LoggerChannelFactoryInterface $logger_channel_factory, MessengerInterface $messenger, AddToQueue $post_queue, Renderer $renderer, FileUrlGeneratorInterface $file_url_generator) {
+  public function __construct(ConfigFactoryInterface $config_factory, EntityTypeManagerInterface $entity_type_manager, LoggerChannelFactoryInterface $logger_channel_factory, MessengerInterface $messenger, AddToQueue $post_queue, Renderer $renderer, FileSystemInterface $file_system, FileRepositoryInterface $file_repository) {
```

Comment:
When they get this long, it is time to slicem and dicem
```suggestion
  public function __construct(
    ConfigFactoryInterface $config_factory,
    EntityTypeManagerInterface $entity_type_manager,
    LoggerChannelFactoryInterface $logger_channel_factory,
    MessengerInterface $messenger,
    AddToQueue $post_queue,
    Renderer $renderer,
    FileSystemInterface $file_system,
    FileRepositoryInterface $file_repository
  ) {
```